ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -61,3 +83,39 @@ func readECSParams(filename string) (*ECSParams, error) {
 
 	return ecsParams, nil
 }
+
+// ConvertToEcsNetworkConfiguration extracts out the NetworkConfiguration from
+// the ECSParams into a format that is compatible with ECSClient calls.
+func ConvertToEcsNetworkConfiguration(ecsParams *ECSParams) (*ecs.NetworkConfiguration, error) {
```

Comment:
nit: go initialism will tell you that "Ecs" should be "ECS" here, and everywhere

Reply:
sigh okay fine